packages/amplify-category-geo/src/service-utils/mapParams.ts

```diff
@@ -61,9 +67,6 @@ export const convertToCompleteMapParams = (partial: Partial<MapParameters>): Map
  * Constructs the Amazon Location Map Style from available map parameters
  */
 export const getGeoMapStyle = (dataProvider: DataProvider, mapStyleType: MapStyleType) => {
-    if (dataProvider === DataProvider.Here && mapStyleType === HereMapStyleType.Berlin) {
-        return MapStyle.VectorHereBerlin;
-    }
     return `Vector${dataProvider}${mapStyleType}`;
```

Comment:
The DataProvider value is `HERE` and hence just `Vector${dataProvider}${mapStyleType}` will result in `VectorHEREBerlin` etc. I would suggest using:
```
if (dataProvider === DataProvider.Here) {
    return `VectorHere${mapStyleType}`;
}
else if (dataProvider === DataProvider.Esri && mapStyleType === EsriMapStyleType.Imagery) {
    return MapStyle.RasterEsriImagery;
}
return `Vector${dataProvider}${mapStyleType}`;
```
This should handle all styles appropriately.

Reply:
This item has been corrected.